tests/basic-test.sh

```diff
@@ -557,8 +557,13 @@ ${CMD_PREFIX} ostree --repo=repo2 pull-local repo
 rm -rf test2-checkout
 ${CMD_PREFIX} ostree --repo=repo2 checkout -U --disable-cache test2 test2-checkout
 if test -d repo2/uncompressed-objects-cache; then
-    assert_file_has_mode test2-checkout/baz/cowro 600
-    assert_file_has_mode test2-checkout/baz/deeper/ohyeahx 755
+    # we're in archive mode, but the repo we pull-local from might be
```

Comment:
No good reason at all. :) I had somehow missed the fact this was an error path. I did push another commit to make this more obvious. ⬇️